src/app/visit/visit.cpp

```diff
@@ -33,6 +33,11 @@ std::vector<Treatment> Visit::getTreatments() const
     return treatments_;
 }
 
+Status Visit::getstatus() const
```

Comment:
Capital letter for S to follow naming convention

Reply:
Done